src/components/Board/Output/Output.container.js

```diff
@@ -184,8 +184,13 @@ export class OutputContainer extends Component {
   handleCopyClick = () => {
     const { intl, showNotification } = this.props;
     const labels = this.props.output.map(symbol => symbol.label);
-    navigator.clipboard.writeText(labels.join(' '));
-    showNotification(intl.formatMessage(messages.copyMessage));
+    try {
+      navigator.clipboard.writeText(labels.join(' '));
```

Comment:
@martinbedouret or @shayc could confirm, but I don't think we should have removed the `await` keyword in d0dc13b6c33f2d7b99c2c83f3ac92ec346202c9f. `writeText` returns a Promise, so if we don't await it, we can't catch the error. This means we will always get a success message and never an error message, even if we fail to write to the clipboard.

@katerinadimatou Instead of removing `await`, you should be able to fix the compilation error by adding `async` to the function definition, like this:

```javascript
handleCopyClick = async () => { ... }
```

Reply:
Obviously it's too late to fix it in this pull request, you would have to create a new one :-)

Reply:
@sylvansson you are correct. the `writeText` method returns a Promise and should be awaited (example)[https://web.dev/async-clipboard/#writetext()]

@katerinadimatou this will require a new PR (small change as @sylvansson suggested)